tests/pipelines/test_contrastive.py

```diff
@@ -0,0 +1,143 @@
+import csv
+from pathlib import Path
+
+from sklearn.ensemble import RandomForestClassifier
+import torch
+import torch.nn.functional as F
+from torch.optim import Adam
+import torchvision
+
+from catalyst import dl
+from catalyst.contrib import nn
+from catalyst.contrib.data.datawrappers import ContrastiveDataset
+from catalyst.contrib.models.cv.encoders import ResnetEncoder
+from catalyst.contrib.nn.criterion import NTXentLoss
+
+
+def read_csv(csv_path: str):
+    with open(csv_path, "r") as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter=",")
+        for line_count, row in enumerate(csv_reader):
+            if line_count == 0:
+                colnames = row
+            else:
+                yield {colname: val for colname, val in zip(colnames, row)}
+
+
+class ContrastiveRunner(dl.SupervisedRunner):
+    def handle_batch(self, batch):
+        # model train/valid step
+        # unpack the batch
+        emb1 = self.model(batch["aug1"])
+        emb2 = self.model(batch["aug2"])
+        self.batch = {"proj1": emb1, "proj2": emb2, "target": batch["target"]}
+
+
+batch_size = 1024
+aug_strength = 1.0
+from torchvision import transforms
+from torchvision.datasets import MNIST
```

Comment:
```suggestion
from catalyst.contrib.datasets import MNIST
```
we could simplify dependencies a bit :)